draft-ietf-gnap-core-protocol.md

```diff
@@ -2688,33 +2688,45 @@ This is often part of facilitating [interaction](#user-interaction), but it coul
 also be used to allow the AS and RC to continue negotiating the parameters of
 the [original grant request](#request). 
 
-To enable this ongoing negotiation, the AS returns a `continue` field 
+To enable this ongoing negotiation, the AS provides a continuation API to the RC.
+The AS returns a `continue` field 
 [in the response](#response-continue) that contains information the RC needs to
-continue this process with another request, including a URI to access
-as well as an optional access token to use during the continued requests.
-
-When the RC makes any calls to the continuation URL, the RC MUST present
-proof of the most recent key associated with this ongoing request
-by signing the request as described in {{binding-keys}}. The key in use will
-be either the key from [the initial request](#request-key) or its most recent
-rotation. 
+access this API, including a URI to access
+as well as an access token to use during the continued requests. 
+
+The access token is initially bound to the same key and method the RC used to make 
+the initial request. As a consequence,
+when the RC makes any calls to the continuation URL, the RC MUST present
+the access token as described in {{use-access-token}} and present
+proof of the RC's key (or its most recent rotation)
+by signing the request as described in {{binding-keys}}.
+
 \[\[ [See issue #85](https://github.com/ietf-wg-gnap/gnap-core-protocol/issues/85) \]\]
 
-For example, here the RC makes a POST request and signs with detached
-JWS:
+For example, here the RC makes a POST request to a unique URI and signs 
+the request with detached JWS:
 
 ~~~
-POST /continue/80UPRY5NM33OMUKMKSKU HTTP/1.1
+POST /continue/KSKUOMUKM HTTP/1.1
+Authorization: GNAP 80UPRY5NM33OMUKMKSKU
 Host: server.example.com
 Detached-JWS: ejy0...
 ~~~
 
-If the AS includes an `access_token` in the `continue`
-response in {{response-continue}}, the RC MUST include the access token the
-request as described in {{use-access-token}}. Note that the access token
-is always bound to the RC's presented key (or its most recent rotation).
+The AS MUST be able to tell from the RC's request which specific ongoing request
+is being accessed. Common methods for facilitating this include using a unique, unguessable URL
+for each continuation response, associating the request with the provided access
+token, or allowing only a single ongoing grant request for a given RC instance
+at a time. If the AS cannot determine a single active grant request to map the
+continuation request to, the AS MUST return an error.
```

Comment:
Proposed alternative text to make this MUST and its guidance more actionable:

```suggestion
The AS MUST be able to tell from the RC's request which specific ongoing request
is being accessed, using a combination of the continuation URL,
the provided access token, and the RC instance identified by the key signature.
If the AS cannot determine a single active grant request to map the
continuation request to, the AS MUST return an error.
```

Reply:
WFM.

Reply:
OK, incorporated.